lib/hexpm/web/views/api/package_view.ex

```diff
@@ -25,15 +25,20 @@ defmodule Hexpm.Web.API.PackageView do
       }
     }
     |> include_if_loaded(:releases, package.releases, ReleaseView, "minimal.json", package: package)
+    |> include_if_loaded(:retirement, package.releases, RetirementView, "package.json")
     |> include_if_loaded(:downloads, package.downloads, DownloadView, "show.json")
     |> include_if_loaded(:owners, package.owners, UserView, "minimal.json")
     |> group_downloads()
+    |> group_retirements()
   end
 
   defp group_downloads(%{downloads: downloads} = package) do
     Map.put(package, :downloads, Enum.reduce(downloads, %{}, &Map.merge(&1, &2)))
   end
-  defp group_downloads(package) do
-    package
+  defp group_downloads(package), do: package
+
+  defp group_retirements(%{retirement: retirement} = package) do
+    Map.put(package, :retirement, Enum.reduce(retirement, %{}, &Map.merge(&1, &2)))
```

Comment:
This should be the plural "retirements" since we can have multiple retirements for one package.

Reply:
![screen shot 2018-01-13 at 9 55 57 am](https://user-images.githubusercontent.com/773380/34908131-1b8e054a-f848-11e7-9176-19db02162919.png)

I have pushed that change.

Reply:
now that i look at it, would it make more sense for retirements to be a list instead of a map?

Reply:
Now that I tried that, the list might not be a great solution since the keys are strings. If we did choose a list, I think the version would have to go back into the map along with the reason and message information.
![screen shot 2018-01-13 at 10 18 44 am](https://user-images.githubusercontent.com/773380/34908354-59c9ea4c-f84b-11e7-96db-cbb5f5d6f70e.png)